wagtail/project_template/home/tests.py

```diff
@@ -0,0 +1,39 @@
+# home/tests.py
+
+from django.test import TestCase
+from wagtail.models import Page
+from home.models import HomePage
+
+
+class HomeSetUpTests(TestCase):
+    """
+    Tests steps needed by follow up tests
+    """
+
+    def test_root_create(self):
+        Page.objects.get(pk=1)
+
+    def test_homepage_create(self):
+        root_page = Page.objects.get(pk=1)
+        self.homepage = HomePage(title='Home')
+        root_page.add_child(instance=self.homepage)
```

Comment:
What is this actually testing?

Do we need some assertions here?

Same for the test above.